pkg/nuctl/command/delete.go

```diff
@@ -43,6 +44,8 @@ func newDeleteCommandeer(ctx context.Context, rootCommandeer *RootCommandeer) *d
 		Short:   "Delete resources",
 	}
 
+	cmd.PersistentFlags().BoolVarP(&commandeer.forceDelete, "force", "f", false, "Force delete resources")
```

Comment:
Maybe it is worth adding a bit more details about what `force delete` is?  It can include something like `it enables cascading deletion strategy`

Reply:
This would mean to move the flag to each commandeer separately, since the strategy is only related to projects and not functions.
If a more descriptive info is needed I'll move it.

Reply:
@TomerShor then why not adding `force` only to `projects` for now? What does this flag change for other `delete`-commanders?

Reply:
It's mostly relevant for deleting a function. See - https://github.com/nuclio/nuclio/pull/3121/files#diff-7ec764da1ad580932d0274b0c2b577cd6f887d85c37d0563cceb7030d84f04d3R99-R101

project deletion already has support for `--deletion-strategy`, but I thought `--force` is more user-friendly and can be aligned with the function flag.

Reply:
I see, okay, then let's leave it as is